moveit_setup_assistant/templates/moveit_config_pkg_template/launch/move_group.launch

```diff
@@ -16,7 +16,7 @@
   <arg name="allow_trajectory_execution" default="true"/>
   <arg name="fake_execution" default="false"/>
   <arg name="execution_type" default="interpolate"/> <!-- set to 'last point' to skip intermediate trajectory in fake execution -->
-  <arg name="max_safe_path_cost" default="1"/>
+  <arg name="sense_for_plan/max_safe_path_cost" default="1"/>
```

Comment:
Indeed, the dynamic reconfigure server for this parameter is placed within the namespace `sense_for_plan`:
https://github.com/ros-planning/moveit/blob/abb83f178075b4d5bb655df2dc9dd3e400e258ca/moveit_ros/planning/plan_execution/src/plan_with_sensing.cpp#L52-L53